analyzers/dataframe/src/myUtils.cc

```diff
@@ -1259,7 +1297,7 @@ ROOT::VecOps::RVec<edm4hep::TrackState> get_pseudotrack(ROOT::VecOps::RVec<Verte
   ROOT::VecOps::RVec<edm4hep::TrackState> result;
   float norm = 1e-3;   // to convert from mm to meters
   for (auto & p: vertex){
-    if (p.vertex.primary>0)continue;
+    if (isPrimaryOrSecondaryVtx(p.vertex))continue;
```

Comment:
This is one of the cases, where I was not sure whether we truly only want the primary vertex, or whether a secondary vertex would also be OK.

Reply:
Analyses using this are:
- flavour/Bs2PhiNuNu
- flavour/B2Kstee
- flavour/Bd2KstNuNu
- flavour/BuBc2TauNu
- flavour/Bc2TauNu

Reply:
@AidanWiederhold, I suppose a similar argument holds for this one as well?

Reply:
Yeah I guess the `DVertex` is in the sense that in flavour we usually look at B->D decays so the `D` vertex would be a secondary vertex, so this is the same as `get_Vertex_thrusthemis_angle` but skipping over the primary vertices

Reply:
Thanks. Also adapted this one to only skip primary vertices.